tools/ci-build/runtime-versioner/src/command.rs

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+use anyhow::{bail, Context, Result};
+use std::process::Command;
+
+/// Extension trait to make `Command` nicer to work with.
+pub trait CommandExt {
+    fn expect_success_output(&mut self, context: &str) -> Result<String>;
+
+    fn expect_statuses(&mut self, statuses: impl IntoIterator<Item = i32>) -> Result<i32>;
+}
+
+impl CommandExt for Command {
+    fn expect_success_output(&mut self, context: &str) -> Result<String> {
+        let output = self
```

Comment:
I feel like we have Command helpers in a few places? is there a common we can put it in? Or a 3p library that does this?

Reply:
We do, but I'm not so happy with them. The previous ones were designed to make unit testing with mocked or faked CLI commands easier, but I've been thinking lately that that's a harder path to take. It's easier to make fake git repositories and integration test against them.

I think the old ones still have a place for the publisher tool though, or at the very least, it would take a non-trivial amount of effort to change them. It's useful to mock out some of the cargo commands for that tool in testing.

I decided to move these extension functions into common and put in a comment noting when one should be used over the other.